docs/guides/run_real_world_car/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,24 @@
+# Troubleshooting
+
+This guide provides solutions for common issues and important checks during operation.
+
+## Missing or Outdated Build Artifacts
+
+If a node fails in execution or there are missing executables and missing libraries.
```

Comment:
I would add to check if the package in question is present in the `~/f1tenth_ws/src` directory. If not, use `ln -s`.

I would also add the error message that would pop up in this case, because that is 90% of the problems

Reply:
What is the error message? I may paste it in the symptoms